libmscore/layout.cpp

```diff
@@ -2543,6 +2543,58 @@ void layoutDrumsetChord(Chord* c, const Drumset* drumset, const StaffType* st, q
             }
       }
 
+//---------------------------------------------------------
+//   extendedStemLenWithTwoNotesTremolo
+//    Extend stem of one of the chords to make the tremolo less steep
+//---------------------------------------------------------
+
+std::pair<qreal, qreal> extendedStemLenWithTwoNoteTremolo(Tremolo* _tremolo, qreal stemLen1, qreal stemLen2)
+      {
+      const qreal _spatium = _tremolo->score()->spatium();
```

Comment:
It’s better to rename `_tremolo` to `tremolo`, otherwise it seems that this is a member of the class  
And `_spatium` too